wiegand/api.py

```diff
@@ -0,0 +1,50 @@
+from RPi.GPIO import *
+import time
+import exceptions
+
+class Wiegand26:
+    # Reads pin numbers for input pins from JSON
+    # Facility number is also set from JSON
+    def __init__(self):
+        self.data0 = 13
+        self.data1 = 15
+        self.facility = 32
+
+    # Sets data pins to IN
+    def setup(self):
+        setwarnings(False)
+        setmode(BOARD)
+        setup(self.data0, IN)
+        setup(self.data1, IN)
+
+    # Loops until Card is detected and returns ID value
+    def read(self):
+        capture = ''
+        while True:
+
+            if input(self.data0) == LOW:
+                capture += '0'
+                time.sleep(.0001)
+
+            elif input(self.data1) == LOW:
+                capture += '1'
+                time.sleep(.0001)
+
+            # Evaluates to true if there are no incoming bits
```

Comment:
are there incoming bits after 26 bits here or are you only looking at 26 bits at a time? from what i understand from this line of code, once `capture` has 26 bits, it will look into the elif condition and do something. but the comment is saying that `there are no incoming bits` so i am not sure if that statement is true.